### PR TITLE
CT-3223 filters for complaint type

### PR DESCRIPTION
**Pull request by ymao2**
## Description

This PR is to add the following filters
1 - Complaint type for open, closed cases, search tab
2 - Complaint priority for open, closed cases, search tab
3 - Complaint subtype for closed cases tab

## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [X] (3) Tests passing
* [X] (4) Branch ready to be merged (not work in progress)
* [X] (5) No superfluous changes in diff
* [X] (6) No TODO's without new ticket numbers
* [X] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`

### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3171

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -380,6 +380,9 @@ en:
         filter_case_type: Type
         filter_open_case_status: Status
         filter_sensitivity: Sensitivity
+        filter_complaint_type: Complaint type
+        filter_complaint_priority: Sensitivity
```
Comment (Polsonby) [inline config/locales/en.yml:384]: I think we've got into a bit of a muddle in the ticket regarding sensitivity and priority. These are two separate things and arguably each requires its own filter options. Sensitivity refers to high profile cases (in the news, etc) whereas priority is a separate field that we introduced for complaints
```suggestion
        filter_complaint_priority: Priority
```

File: config/locales/filters.en.yml [hunk 2]
```diff
@@ -39,6 +45,9 @@ en:
       filter_date_responded: Filter by date of case response
       filter_exemption: Filter by exemptions
       filter_sensitivity: Filter by sensitivity
+      filter_complaint_type: Filter by complaint ype
```
Comment (Polsonby) [inline config/locales/filters.en.yml:48]: missing letter ;) 

```suggestion
      filter_complaint_type: Filter by complaint type
```

Comment (ymao2) [reply config/locales/filters.en.yml:48]: @Polsonby just looked at the ticket again, yes, I think I misunderstood the sensitivity,  let me change it

**Review by Polsonby — APPROVED** [review]
LGTM :) great work

**Pull request merged**